### PR TITLE
update dependencies to wicket 7.0.0 stable

### PR DESCRIPTION
**Pull request by BeneStem**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ org.eclipse.jdt.apt.core.prefs
 org.eclipse.m2e.core.prefs
 org.eclipse.pde.core.prefs
 
+wiquery-*.i*
```
Comment (martin-g) [inline .gitignore]: What is this ?

**Comment by martin-g** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (BeneStem) [inline .gitignore]: I use IntelliJ for development.
And it creates some files with .ipr, .iml and .iws ending.
Just didn't want to push them.
You can revert this particular change if you want to :)

**Comment by BeneStem** [comment]
Thanks!
We are not sure if everything works by now.
There seems to be a problem with resource caching...
Some times wicket is not able to load the i18n js file...
We are currently looking into it and will perhaps create another pull request :)

File: .gitignore (hunk 1 above)
Comment (martin-g) [inline .gitignore]: It is OK!

Comment (BeneStem) [inline .gitignore]: :+1:

**Comment by martin-g** [comment]
There were some improvements in Wicket 7.1.0-SNAPSHOT related to that. Try with latest Wicket from `wicket-7.x` branch before spending time in debugging.

**Comment by BeneStem** [comment]
Thanks for the hint.
We will give it a try tomorrow morning (it is 22:10 in Germany right now)

**Comment by BeneStem** [comment]
Thanks for the hint again.
We tried the 7.1.0-SNAPSHOT and it did the trick.
Everything works fine now.
So this pull request is all it needs to get wiquery to 7.x!

**Comment by martin-g** [comment]
Thanks for the contribution and for testing Wicket Snapshot!

**Comment by martin-g** [comment]
@papegaaij @hielkehoeve @dashorst @reiern70 Who could make a release ?
I'd do it but I think I don't permissions for Sonatype OSS.

**Comment by BeneStem** [comment]
Not a problem at all.
We are working with wicket for about four years now.
Thank you for your great support!
Looking forward to a 7.1.X stable wicket release ;)

**Comment by dashorst** [comment]
I think only @hielkehoeve has permissions of us (not sure whether @reiern70 has release rights)

**Comment by martin-g** [comment]
@hielkehoeve Would you mind add me to the team at Sonatype OSS so I can help with these tasks? Thanks!

**Comment by martin-g** [comment]
Here is what we gonna do:
*) Move WiQuery project under https://github.com/wicketstuff/, but **not** under https://github.com/wicketstuff/core!
*) Change the groupId to *org.wicketstuff.wiquery*
*) Release 7.0.0
*) We will release a new version when there are changes in the code

It won't be included in WicketStuff Core because there is no need to release it regularly if there are no changes.

I won't be able to do this until next week. If anyone wants to help please send a PR!

**Comment by BeneStem** [comment]
Great idea!
It really fits as a wicketstuff addon.

**Comment by martin-g** [comment]
The repo has been moved to https://github.com/wicketstuff/wiquery (thanks, Martijjn!).
I've just tested my current client's app with wicket-6 branch. I'll try to migrate the app to Wicket 7.x in the coming days and test wicket-7 branch.
Please build wicket-7 branch locally and test it so we can make a release sooner!
Thanks!

**Comment by BeneStem** [comment]
Hey guys!
I will build it locally and add it to our application tomorrow. (Wicket 7)
After that i will post feedback here ;)
Thank you for the migration to wicketstuff!

**Comment by BeneStem** [comment]
omg, 24 days -.-
sorry guys!
i am on it now :)
too much things to do...

**Comment by martin-g** [comment]
It is released and I even upgraded one of our apps to use it :-)
Please report any issues to the new repo.
Thank you!

**Comment by BeneStem** [comment]
you are the best!
and i was way too slow :D

**Pull request merged**